### ISSUE TITLE
HTML entities as inline style (font family) not replaced correctly

### ISSUE THREAD

**Issue by dharmendrasdlt** [tdewolff/minify#272]
Reproducible snippet is here: https://play.golang.org/p/XmaVUSnS1rt
Input:
```
<div style="font-family: Arial, &#39;sans-serif&#39;; font-size: 22px;">Golang</div>
```

Output:
```
<div style="font-family:Arial,'sans-serif'font-size:22px">Golang</div>
```
During minification, It replaces `&#39;;` with `'`. Instead it should just replace it like `';`.

Regarding html entity `&#39;` in this example, It is output of x/net/html (https://godoc.org/golang.org/x/net/html) from `html.Render()`

**Comment by tdewolff** [comment]
Fixed in aa9bafa69e9c747799534f775670b6d5ecd61ed0 and 200330f7484a1ad6b3b150cace33e821688e117d, thanks!

**Comment by pierrre** [comment]
thank you ! 👍
